events/fair-ml.html

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: "Fair ML Speaker Series"
+permalink: "/fair-ml"
+---
+
+<h1 class="title page-title"> Fair ML Speaker Series</h1>
+
+<p style = "font-size: 115%;">
+	Thanks to our <a class="tla-link" href="{{site.baseurl}}/speaker-series"> Speaker Series Team </a>, our Fair ML Speaker Series was a success! <b> Learn all about it on this page. </b>
+
+</p>
+
+<hr class="divider" />
+
+<div class="citylab-section" style = "margin-top: 2em;">
+	<div class = "grid-col">
+		<img class = "img-responsive" src = "{{site.baseurl}}/img/undraw/content.svg" alt = "drawing related to content" />
+	</div>
+	<div>
+		<h2 class = "text-2x"> Why does fair machine learning matter?
+		</h2>
+		<p style = "font-size: 115%;">
+			Have you ever wondered... Can AI discriminate? Of course! Thanks to Catherine Yeo, we were inspired to teach others about fair machine learning. Check out her work below to learn more.
+		</p>
+
+		<ul style = "font-size: 150%;">
+			<li> <a class="tla-link" href="http://catherineyeo.tech/ai_fairness_course" target="_blank" rel = "noopener noreferrer">Fairness in AI and Algorithms Class Syllabus </a>
+			</li>
+			<li> <a class = "tla-link" href="https://slack-files.com/files-pri-safe/TA3824MMX-F01BCVAJZ6X/fairness-ethics-in-ai-talk.pdf?c=1607066003-e8f6a1cc96e00768" target = "_blank" rel = "noopener noreferrer"> Catherine's Slides </a>
+			</li>
+			<li> <a class = "tla-link" href="https://medium.com/fair-bytes" target = "_blank" rel = "noopener noreferrer"> Fair Bytes </a>
+			</li>
+			</li>
+		</ul>
+	</div>
+</div>
+
+<div class = "citylab-section citylab-section-reverse" style = "margin-bottom: 3em;">
+	<div class = "grid-col">
+		<img class = "img-responsive" src = "{{site.baseurl}}/img/undraw/predpol.svg" alt = "drawing related to AI" />
+	</div>
+<div>
+	<h2 class="title text-2x">
+		Predictive Policing
+	</h2>
+	<p style = "font-size: 115%;">
+		So you know AI can be biased, but how? Predictive policing uses ML and AI to try to predict crime. Often, the data used is skewed by human bias and inconsistent crime reports. Want to learn more about predictive policing? We got you covered!
+	</p>
+
+	<div class = "button-container">
+		<a class="button button-dark" href="
+		https://docs.google.com/presentation/d/1IRvK3awKEll42cs8MTc3YdCONHa4E_3hV-dU2inlGQ8/edit#slide=id.g9fdeae80bb_0_18" target ="_blank" rel = "noopener noreferrer"><span class = "fa fa-link"></span> Link to Case Study</a>
```

Comment:
Nitpicky, but a different Icon here might be nice! Something that feels more "presentation"/"case-study"-ish. You can find the list of free font awesome icons at [the FA5 website](https://fontawesome.com/icons?d=gallery)